lib/maintenance_tasks/engine.rb

```diff
@@ -21,6 +21,11 @@ class Engine < ::Rails::Engine
       MaintenanceTasks.backtrace_cleaner = Rails.backtrace_cleaner
     end
 
+    initializer "maintenance_tasks.assets.precompile" do |app|
+      app.config.assets.precompile << "maintenance_tasks.js"
```

Comment:
>I'm pretty sure we can make direct uploads work with a configuration option while letting the application deal with assets, especially since direct uploads come from Active Storage.

The issue I ran into was trying to make it include the correct version of `activestorage.js`, matching the rails' version, and have an integrity check for it. The simplest solution was to use the version already provided locally instead of importing one from a public CDN.

Do you have a better suggestion?